apache/metron
# METRON-1869 Unable to Sort an Escalated Meta Alert

This fixes a bug that causes meta-alerts to not be visible in the Alerts UI when the UI is sorted by 'alert_status' and a meta-alert has been escalated.  This is only a problem when indexing into Elasticsearch.

The root cause is that the 'alert_status' field needs to be defined as a keyword in the metaalert index template. This field only exists in the meta-alert index after a meta-alert has changed status, like when an alert is escalated.

## Changes

* Fixed the Elasticsearch meta-alerts template that is deployed with the MPack.
* Added an integration test for this bug.

## Testing

Follow the "steps to reproduce" outlined in the JIRA to ensure that the bug has been squashed.

1. Create a meta-alert.
2. Escalate the meta-alert.
3. Submit another search that filters the results to only show meta-alerts.
4. Sort the results by “alert_status” field.
5. The meta-alert should still be visible.

## Pull Request Checklist
